docs/source/deployment/databricks.md

```diff
@@ -223,3 +223,71 @@ Out[12]: {}
 Your complete notebook should look similar to this (the results are hidden):
 
 ![](../meta/images/databricks_notebook_example.png)
+
+
+### 9. Using the Kedro IPython Extension 
+
+You can interact with Kedro in Databricks through the Kedro [IPython extension](https://ipython.readthedocs.io/en/stable/config/extensions/index.html), `kedro.extras.extensions.ipython`.
+
+The Kedro IPython extension launches a [Kedro session](../kedro_project_setup/session.md) and makes available the useful Kedro variables `catalog`, `context`, `pipelines` and `session`. It also provides the `%reload_kedro` [line magic](https://ipython.readthedocs.io/en/stable/interactive/magics.html) that reloads these variables (for example, if you need to update `catalog` following changes to your Data Catalog).
+
+The IPython extension can be used in a Databricks notebook in a similar way to how it is used in [Jupyter notebooks](https://kedro.readthedocs.io/en/0.18.0/tools_integration/ipython.html).
+
+Currently in Databricks, when you try to do a kedro run, it throws you a `ContextualVersionConflictError` because it downgrades `pip` to 21.0.1. We are not sure why this happens and hence there's one additional step you need to do in the Databricks notebook to make use of the IPython extension. After you load the IPython extension using the below command:
+
+```ipython
+In [1]: %load_ext kedro.extras.extensions.ipython
+```
+
+You must explicitly upgrade your `pip` version by doing the below so it is on the latest version `pip ~= 22.0.4`. 
```

Comment:
Let's leave out the specific version number so we don't have to keep updating this...
```suggestion
You must explicitly upgrade your `pip` version by doing the below: 
```